www/docs/alert/index.php

```diff
@@ -341,6 +341,38 @@ function display_search_form ( $alert, $details = array(), $errors = array() ) {
             echo '<em class="error">You have used a comma in your search term &ndash; are you sure this is what you want?
 You cannot sign up to multiple search terms using a comma &ndash; either use OR, or fill in this form multiple times.</em>';
         }
+        if ( preg_match('#([A-Z]{1,2}\d+ ?\d[A-Z]{2})#i', $alertsearch, $m) && strlen($alertsearch) > strlen($m[1]) && validate_postcode($m[1]) ) {
```

Comment:
This won't cover all postcodes; could we alter `validate_postcode` to optionally allow within string as well as whole string as it currently is?